KUBEVIRTCI_LOCAL_TESTING.md

```diff
@@ -0,0 +1,49 @@
+# [kubevirtci](README.md): Testing kubevirtci with locally provisioned cluster
+
+After making changes to a kubevirtci provider, you should test it locally before publishing it.
+
+Let's go on the required steps, starting by provisioning, all the way to `make cluster-up`.
+Note: the `...` in the beginning of `.../directory_name` means the full path of `directory_name`.
```

Comment:
Has `...` commonly used elsewhere? To me using `cd $KUBEVIRT_CI_DIR` would be more readable.